libs/vertexai/pyproject.toml

```diff
@@ -17,6 +17,11 @@ google-cloud-aiplatform = "^1.47.0"
 google-cloud-storage = "^2.14.0"
 # optional dependencies
 anthropic = { extras = ["vertexai"], version = ">=0.23.0,<1", optional = true }
+pandas = { version = ">= 1.0.0", optional = true }
+google-cloud-bigquery-storage = { version = ">=2.6.0,<3",  optional = true }
+pyarrow = { version = ">= 6.0.1", optional = true }
+langchain-community = { version = ">=0.0.28", optional = true }
+db-dtypes = { version = "^1.2.0", optional = true }
```

Comment:
should this be in the test group?

Reply:
no it's required for pandas - BQ interaction